mkt/zadmin/templates/admin/base.html

```diff
@@ -73,15 +73,6 @@
                       </li>
                     </ul>
                   </li>
-                {% else %}
```

Comment:
Okay, the pages are wrapped in @require_admin decorators or similar I guess.  It would only be useful if there were any public /admin pages (e.g. status pages) but I think they're in /monitor